src/js/index.js

```diff
@@ -13,6 +13,74 @@ const navBookmark = document.querySelector('[data-js=navBookmark]')
 const navCreate = document.querySelector('[data-js=navCreate]')
 const navSettings = document.querySelector('[data-js=navSettings]')
 
+// Get main sections (^= pages)
```

Comment:
I like how you try to guide us with that comments. We will aim for self explanatory code that does not need as many comments in the future. Stay tuned!